.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Report a bug
+about: File a bug to help us improve ML.NET
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**System Information (please complete the following information):**
+ - OS & Version: [e.g. Windows 10] 
+ - ML.NET Version : [e.g. ML.NET v1.5.5]
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
It may be good to list here either code or sample project upload/etc. We usually have to ask for that anyways. It is implied by the steps to reproduce, but may be good to explicitly call out.

Reply:
Fixed by https://github.com/dotnet/machinelearning/pull/5846/commits/4fefd4efb8f06177257842bc852034acf273aaee